client_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/alicebob/miniredis"
-	"github.com/elliotchance/redismock"
+	"github.com/vearne/redismock"
```

Comment:
Tip: This means you have the package cloned in the wrong location - you shouldn't need to change imports.